### ISSUE TITLE
Improve create experience on subscription with limited permissions

### ISSUE THREAD

**Issue by jjmoyaSMFR** [microsoft/vscode-azureappservice#1341]
Does this occur consistently? Yes
Repro steps:
I followed the steps for deployment located here:

https://docs.microsoft.com/en-us/azure/app-service/app-service-web-get-started-nodejs

Action: appService.Deploy
Error type: 403
Error Message: Failed with code "403".

Version: 0.16.2
OS: win32
OS Release: 10.0.17134
Product: Visual Studio Code
Product Version: 1.41.1
Language: en

<details>
<summary>Call Stack</summary>

```
extension.bundle.js:218:501748extension.bundle.js:218:501748
s extension.bundle.js:218:111699
s extension.bundle.js:157:26069
extension.bundle.js:157:27311extension.bundle.js:157:27311
o extension.bundle.js:164:7076
extension.bundle.js:157:24982extension.bundle.js:157:24982
$._callback extension.bundle.js:1:580031
$.t._callback.t.callback.t.callback [as callback] extension.bundle.js:194:5449
$.emit events.js:200:13
$.<anonymous> extension.bundle.js:194:18309
$.emit events.js:205:15
IncomingMessage.<anonymous> extension.bundle.js:194:17141
Object.onceWrapper events.js:288:20
IncomingMessage.emit events.js:205:15
endReadableNT _stream_readable.js:1154:12
processTicksAndRejections task_queues.js:84:9
```

</details>

**Comment by EricJizbaMSFT** [comment]
@jjmoyaSMFR as a workaround, please try the "Advanced" option when it asks you to create a web app. Your subscription probably doesn't have permissions for something.

Related to https://github.com/microsoft/vscode-azurefunctions/issues/1583

**Comment by jjmoyaSMFR** [comment]
Hi Eric,

Thanks for your response.  I have since solved the issue, sort of.  I could not get my app to work in Azure when I was developing and deploying it using Visual Studio Code.  However, when I recreated my app as a visual studio project it worked great.  I guess Studio has better integration with Azure than Studio Code.

James J Moya, MS, MBA
Programmer/Analyst
South Metro Fire Rescue
Information Technology Department
720-989-2447
